docs/rfcs/XeTile.md

```diff
@@ -91,17 +144,32 @@ A `tile_mma` variant without vector_c initialization.
 		tile<64x64xbf16>, index, index into tile <64x64xbf16>
 ```
 
+`tile_pack` packs a 2D vector, representing the loaded value from 2D tile, to a 4D vector with an inner block size. The 4D vector was introduced to support blocking to fit the hardware matrix operation sizes.  The blocking follows an implicit rule: dim[0] is split to dim[0] and dim[2], and dim[1] is split to dim[1] and dim[3]. The dim[2] and dim[3] of result 4D vector must be same as the size of `inner_blocks` attribute. 
```

Comment:
.. an implicit rule: `out_dim[0] = in_dim[0]/inner_blocks[0] , out_dim[1] = in_dim[1]/inner_blocks[1], out_dim[2] = inner_blocks[0], and out_dim[3] = inner_blocks[1]`